e2e/setup.ts

```diff
@@ -20,70 +21,164 @@ export const setup = async () => {
   const db = drizzle(
     postgres("postgresql://postgres:secret@127.0.0.1:5432/postgres"),
   );
+
   const addE2EArticleAndComment = async (
     authorId: string,
     commenterId: string,
   ) => {
     const publishedPostId = nanoid(8);
     const scheduledPostId = nanoid(8);
     const draftPostId = nanoid(8);
-    const now = new Date().toISOString();
 
-    const oneYearFromToday = new Date(now);
-    oneYearFromToday.setFullYear(oneYearFromToday.getFullYear() + 1);
-
-    await Promise.all([
-      db
-        .insert(post)
-        .values({
-          id: publishedPostId,
-          published: now,
-          excerpt: articleExcerpt,
-          updatedAt: now,
-          slug: "e2e-test-slug-published",
-          likes: 10,
-          readTimeMins: 3,
-          title: "Published Article",
-          body: articleContent,
-          userId: authorId,
-        })
-        .onConflictDoNothing()
-        .returning(),
+    const now = new Date().toISOString();
+    const scheduled = new Date(
+      new Date().setFullYear(new Date().getFullYear() + 1),
+    ).toISOString();
 
-      db
-        .insert(post)
-        .values({
-          id: draftPostId,
-          published: null,
-          excerpt: articleExcerpt,
-          updatedAt: now,
-          slug: "e2e-test-slug-draft",
-          likes: 10,
-          readTimeMins: 3,
-          title: "Draft Article",
-          body: articleContent,
-          userId: authorId,
-        })
-        .onConflictDoNothing()
-        .returning(),
+    const articles: Article[] = [
```

Comment:
Check out the seed file in how we can generate data consistently. Will be a good follow up pr